opentracing/span.py

```diff
@@ -63,18 +62,17 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                                             'tb': exc_tb})
         self.finish()
 
-    def start_child(self, operation_name):
+    def start_child(self, operation_name, tags=None):
```

Comment:
stating the obvious here, but there's both

```
  child_span = parent_span.start_child("op").add_tags(tags)
```

and

```
  child_span = parent_span.start_child("op", tags)
```

Since it's easier to add surface area than remove surface area, I'd vote for keeping the former (more general) and removing the latter.